explorer_frontend/src/features/account-connector/components/EndpointScreen.tsx

```diff
@@ -99,6 +102,12 @@ const EndpointScreen = () => {
     }
   }, [smartAccount, balance, balanceToken]);
 
+  useEffect(() => {
```

Comment:
What about providing $endpoint as a default value here?
`const [inputValue, setInputValue] = useState("");`

Reply:
Much better, you are right. Fixed